src/accessories/characteristics/currentPosition.ts

```diff
@@ -15,43 +16,10 @@ export class CurrentPositionCharacteristic extends TuyaWebCharacteristic {
   }
 
   public getRemoteValue(callback: CharacteristicGetCallback): void {
-    this.accessory
-      .getDeviceState()
-      .then((data) => {
-        this.debug("[GET] %s", data?.state);
-        this.updateValue(data, callback);
-      })
-      .catch(this.accessory.handleError("GET", callback));
+    this.updateValue({}, callback);
```

Comment:
Please revert the changes here so that remove changes are reflect properly i.e. the state is opening or closing (i.e. motor active) when the API is currently reporting this state.